addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11570,16 +11570,22 @@ msgstr ""
 #. label for "music/ballet/dance" epg subgenre value
 #: xbmc/pvr/epg/Epg.cpp
 msgctxt "#19584"
-msgid "Musical / Opera"
+msgid "Jazz"
 msgstr ""
 
 #. label for "music/ballet/dance" epg subgenre value
 #: xbmc/pvr/epg/Epg.cpp
 msgctxt "#19585"
+msgid "Musical / Opera"
+msgstr ""
+
+#. label for "music/ballet/dance" epg subgenre value
+#: xbmc/pvr/epg/Epg.cpp
+msgctxt "#19586"
 msgid "Ballet"
 msgstr ""
 
-#empty strings from id 19586 to 19595
+#empty strings from id 19587 to 19595
```

Comment:
@ksooo Maybe it would be good to add a note on begins and declare as number group and that the string numbers relates to EPG number about select so that not wrongly becomes new other strings added (where maybe extended on EPG in future)?

E.g. here: `#empty reserved strings from id 19587 to 19595 about sub genre "music/ballet/dance"`

And on others e.g. `#. label for "music/ballet/dance" epg genre value, subgenres in string id "19580 + sub genre id"` (know my english is bad :sweat_smile:, only to see what I mean).

Reply:
Not sure those `#empty strings from`lines are auto-generated and/or modifying it manually would actually work.